processing/audio_analysis.py

```diff
@@ -0,0 +1,47 @@
+import json
+import logging
+import mimetypes
+import subprocess
+import tempfile
+from pathlib import Path
+
+import processing
+
+
+def process(recording, conf):
+    logger = processing.logs.worker_logger("audio.analysis", recording["id"])
+
+    api = processing.API(conf.api_url)
+    s3 = processing.S3(conf)
+
+    input_extension = mimetypes.guess_extension(recording["rawMimeType"])
+
+    if not input_extension:
+        # Unsupported mimetype. If needed more mimetypes can be added above.
+        logger.error("unsupported mimetype. Not processing")
+        api.report_done(recording, recording["rawFileKey"], recording["rawMimeType"])
```

Comment:
will this set anything in the database to represent it failed?

Reply:
Yes there is and there's #7 to actually take advantage of it.